js/main.js

```diff
@@ -11,33 +19,53 @@ const finalResult = document.querySelector(".equal-sign");
 finalResult.addEventListener("click", tallyResult)
 
 function addToDisplay(e) {
-    let outcome = document.querySelector("#beep");
-    let itemClicked = e.path[0].innerHTML;
+    const outcome = document.querySelector("#beep");
+    const itemClicked = e.path[0].innerHTML;
     if (itemClicked <= 9) {
-        let itemNumber = Number(itemClicked);
+        const itemNumber = Number(itemClicked);
         outcome.innerHTML += itemNumber;
     } else if (itemClicked === "/" || itemClicked === "*" || itemClicked === "-" || itemClicked === "+") {
-        outcome.innerHTML += " " + itemClicked + " ";
+        // BK Time to learn about template strings!
+        outcome.innerHTML += ` ${itemClicked} `;
     }
-   let myParts =  outcome.innerHTML.split(" ")
+   const myParts = outcome.innerHTML.split(" ")
    return myParts;
 
 }
 
 //When user clicks the "=", do math of what is in the h2, and then re assign the h2 to that new value.
-
+/**
+ * BK - Setting the h2 is fine in vanilla JS- in a framework you would assign a variable in the component to the end value and call that variable in the DOM.
+ * Again, fine for now, but just for future reference
+ */
 function tallyResult() {
-    let outcome = document.querySelector("#beep");
-    let myParts = outcome.innerHTML.split(" ")
+    const outcome = document.querySelector("#beep");
+    /**
+     * BK - array is a good choice here as you have done. That would potentially allow you to do multiple operations
+     * Consider: "1 + 2 * 3"
+     * Operators (+, -, etc) are in odd places, numbers in even
+     * or in JS speak, for i in myParts, if i % 2 === 0, it is a number
+     * You'd then have to deal with order of operations- e.g. * happens before +
+     * 
+     * This if else statement isn't bad or wrong necessarily, but I'm going to revamp it a bit to show you some object oriented ideas that can be really handy when you get into longer chains-
+     * what if this had 10 else ifs? Would be kinda gnarly looking
+     */
+    const ops = {
+        '+': (first, second) => first + second,
+        '-': (first, second) => first - second,
+        '*': (first, second) => first * second,
+        '/': (first, second) => first / second
+    };
```

Comment:
I really like this idea.  Objects are still a little new to me, I haven't totally grasped the concept yet.  It does look cleaner than if/else.